package.json

```diff
@@ -9,9 +9,6 @@
 		"email": "sindresorhus@gmail.com",
 		"url": "sindresorhus.com"
 	},
-	"engines": {
-		"node": ">=6"
-	},
```

Comment:
Don't do unrelated changes.

Reply:
Good catch.
That should not have made it in here. I will close this and re-open from a diff SHA. 
No one wants a dirty git history.